x/launchpad/keeper/project_services.go

```diff
@@ -12,6 +15,23 @@ func validateCreateProjectMsg(ctx sdk.Context, msg *types.MsgCreateProjectReques
 	return nil
 }
 
+func validateModifyInformationMsg(ctx sdk.Context, msg *types.MsgModifyProjectInformationRequest) error {
+	return nil
+}
+
+func validateModifyStartTimeMsg(ctx sdk.Context, msg *types.MsgModifyStartTimeRequest) error {
```

Comment:
please change this to validateProjectActive().

In the original design, a modification is allowed if a project is not active "ProjectActive" field of Project struct. Not based on whether it has passed StartTime.